build.gradle

```diff
@@ -32,8 +32,8 @@ dependencies {
   compile group: 'org.jenkinsci.plugins', name: 'pipeline-model-definition', version: '1.3.7', ext: 'jar'
   compile 'org.jenkins-ci.plugins:job-dsl-core:1.65'
 
-  compile 'com.microsoft.azure:azure-documentdb:1.15.2'
-  compile 'com.microsoft.azure:azure:1.10.0'
+  compile 'com.microsoft.azure:azure-documentdb:2.4.1'
+  compile 'com.microsoft.azure:azure:1.29.0'
```

Comment:
The build.gradle file is a bit misleading - these deps are only used during the local 'build' and tests.

When running on Jenkins the actual deps are grabbed using grapes (i hate groovy naming) https://github.com/hmcts/cnp-jenkins-library/search?q=Grab&unscoped_q=Grab

Reply:
http://docs.groovy-lang.org/latest/html/documentation/grape.html